packages/ag-charts-community/src/chart/series/sankey/sankeySeriesModule.ts

```diff
@@ -0,0 +1,19 @@
+import type { SeriesModule } from '../../../module/coreModules';
+import { EXTENDS_SERIES_DEFAULTS } from '../../themes/symbols';
+import { SankeySeries } from './sankeySeries';
+
+export const SankeySeriesModule: SeriesModule<'sankey'> = {
+    type: 'series',
+    optionsKey: 'series[]',
+    packageType: 'community',
```

Comment:
This should probably be an enterprise package type - so we might need to hoist some of these files into the enterprise package.

Reply:
Yep - it’s easier to do development on community packages, so I’ll move it to enterprise when it’s more ready (like maps)